frontend/src/metabase-lib/metrics.ts

```diff
@@ -1,7 +1,15 @@
 import * as ML from "cljs/metabase.lib.js";
+import type { MetricId } from "metabase-types/api";
 
 import type { MetricMetadata, Query } from "./types";
 
 export function availableMetrics(query: Query): MetricMetadata[] {
   return ML.available_metrics(query);
 }
+
+export function metric(
+  query: Query,
+  metricId: MetricId,
+): MetricMetadata | null {
+  return ML.metric(query, metricId);
```

Comment:
`metric_metadata` to match `table_or_card_metadata`?

Reply:
8c8769650092609cc5a373d976255eb6d4ad5bce